packages/app/src/pages/Specs/Index.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div v-if="query.data.value">
+  <div
+    v-if="query.data.value"
+    class="h-full"
```

Comment:
I know there's some *very* finicky styling around the Specs List to support the virtualized list - may want to validate it's still behaving as expected after this change (and in `SpecsList.vue`)